lib/global_registry/base.rb

```diff
@@ -81,18 +81,25 @@ def request(method, params, path = nil, headers = {})
       case method
       when :post
         post_headers = default_headers.merge(content_type: :json, timeout: nil).merge(headers)
-        RestClient.post(url.to_s, params.to_json, post_headers) { |response, request, result, &block|
+        post_args = { method: method, url: url.to_s, timeout: nil, params: params.to_json,
+                      headers: post_headers, proxy: GlobalRegistry.proxy_url
```

Comment:
Will RestClient react ok if `proxy_url` is not set and this param is nil?

Reply:
Yes it will. We have tested this exact case.

Look here in the Rest Client docs: https://github.com/rest-client/rest-client#proxy.
Specifically the code where they show this set of lines:
```
RestClient.proxy = "http://proxy.example.com/"
RestClient::Request.execute(method: :get, url: 'http://example.com', proxy: nil)
# => single request sent without a proxy
```